.codecov.yml

```diff
@@ -0,0 +1,35 @@
+codecov:
+  # Don't wait for all other statuses to pass before sending codecov status.
+  require_ci_to_pass: no
+
+# See https://docs.codecov.com/docs/pull-request-comments
+comment:
+  layout: " diff, files"
+  behavior: default # Update if exists, otherwise post new.
+  require_changes: false  # post the comment even when no coverage changes
+
+# See https://docs.codecov.com/docs/commit-status
+coverage:
+  status:
+    project:
+      default:
+        target: auto
+        threshold: 5%
```

Comment:
This value or even the target of the patch can be discussed later on as we experiment with the current configuration

Reply:
This should be a project variable that we modify globally without making a PR.

Reply:
You mean the https://docs.github.com/en/actions/learn-github-actions/variables? AFAIK they are meant to be used in workflows and this configuration happens from a YAML file in the repo. We could somehow "modify" the file with this value in each CI run but it sounds a bit cumbersome and in that case I'd tackle it in a following PR